src/region.js

```diff
@@ -42,151 +41,145 @@ var Region = MarionetteObject.extend({
   // the old view being destroyed on show.
   // The `forceShow` option can be used to force a view to be
   // re-rendered if it's already shown in the region.
-  show: function(view, options) {
+  show: function(view, options={}) {
     if (!this._ensureElement()) {
       return;
     }
 
-    this._ensureViewIsIntact(view);
```

Comment:
Only that show() got a good bit shorter, so I unrolled some methods back into show() for ease of following the logic.  However just throwing exceptions isn't core method logic, so I agree with keeping `_ensureViewIsIntact()`.